install/deploy_management_nodes.md

```diff
@@ -317,6 +317,14 @@ be performed are in the [Deploy](#deploy) section.
     This will create folders for each host in `/var/www`, allowing each host to have its own unique set of artifacts:
     kernel, `initrd`, SquashFS, and `script.ipxe` bootscript.
 
+    1. Patch the `set-sqfs-links.sh` script to include the blacklisting of an undesired kernel module.
+
+    ```bash
+    pit# sed -i -E 's/rd.luks=0/rd.luks=0 module_blacklist=rpcrdma/g' /root/bin/set-sqfs-links.sh
```

Comment:
We should consider making a release branch or pit-init as to not hack the script in the docs.

Reply:
I definitely agree there should be release branches (or tags or whatever) of pit-init.

But there will still always be the risk of making these edits in the docs in cases where they don't want to build a new image.

Reply:
Yeah, the idea of spinning a new LiveCD was nixed during the standup today.